services/catalog/src/simcore_service_catalog/db/repositories/projects.py

```diff
@@ -17,7 +17,7 @@ async def list_services_from_published_templates(self) -> list[ServiceKeyVersion
             async for row in await conn.stream(
                 sa.select(projects).where(
                     (projects.c.type == ProjectType.TEMPLATE)
-                    & (projects.c.published == True)
+                    & (projects.c.published is True)
```

Comment:
is this legal? should you not use ```.is_(True)``` ??

Reply:
Goot catch ... ! I believe it was rust changes ... but not sure.
The question here is why the tests did not catch that! Will also increase coverage a bit more before start doing changes

Reply:
@sanderegg FYI:
```cmd
>>> from simcore_postgres_database.models.projects import projects
>>> from sqlalchemy.dialects import postgresql
>>> import sqlalchemy as sa
```
then 
```cmd
>>> stmt = sa.select(projects).where(projects.c.published is True)
>>> print(stmt.compile(dialect=postgresql.dialect(), compile_kwargs={"literal_binds": True}))
SELECT projects.id, projects.type, projects.uuid, projects.name, projects.description, projects.thumbnail, projects.prj_owner, projects.creation_date, projects.last_change_date, projects.access_rights, projects.workbench, projects.ui, projects.classifiers, projects.dev, projects.quality, projects.published, projects.hidden 
FROM projects 
WHERE false
```
vs
```
>>> stmt = sa.select(projects).where(projects.c.published.is_(True))
>>> print(stmt.compile(dialect=postgresql.dialect(), compile_kwargs={"literal_binds": True}))
SELECT projects.id, projects.type, projects.uuid, projects.name, projects.description, projects.thumbnail, projects.prj_owner, projects.creation_date, projects.last_change_date, projects.access_rights, projects.workbench, projects.ui, projects.classifiers, projects.dev, projects.quality, projects.published, projects.hidden 
FROM projects 
WHERE projects.published IS true
```